docs/source/using-discovery/slurm.md

```diff
@@ -0,0 +1,769 @@
+
+Slurm allows users to submit their computational tasks as jobs to be scheduled on the cluster's compute nodes. Its role-based access control ensures proper resource allocation and job execution, preventing resource conflicts.
+
+Slurm is crucial in research environments, where it ensures fair usage of resources among a multitude of users, helps optimize the workload for the available resources, and provides precise job accounting and statistics.
+
+**Page Objective:**
+
+This tutorial aims to equip users with an understanding of the Slurm workload manager. It starts with the basics, teaching you how to install and configure Slurm, before diving into the more nuanced aspects of job submission and management. The tutorial also covers advanced usage, troubleshooting, and best practices to get the most out of your HPC environment.
+
+**Who Should Use This Guide?**
+
+This guide is ideal for beginners new to Slurm and HPC, researchers intending to use Slurm-based clusters for their computation tasks, system administrators managing HPC environments, and even seasoned HPC users looking to brush up on their knowledge. It progresses from fundamental to advanced topics, making it a valuable resource for a broad audience.
+
+## Slurm: Basic Concepts
+Before we delve into using Slurm, it's essential to grasp some fundamental concepts related to its operation.
+
+### Nodes
+In the context of Slurm, a 'node' refers to a server within the HPC cluster. Each node possesses its resources, such as CPUs, memory, storage, and potentially GPUs. Slurm manages these nodes and allocates resources to the tasks.
+
+(slurm-partitions)=
+### Partition(s)
+A 'partition' is a grouping of nodes. You can think of partitions as virtual clusters within your HPC system. They allow administrators to segregate the compute environment based on factors like job sizes, hardware type, or resource allocation policies.
+
+:::{seealso}
+Our {ref}`partition-names` documentation.
+:::
+
+### Account information
+Some Discovery users have more than one Discovery group account associated with their usernames. For example, a student might be in a class using Discovery and a student club using Discovery for a club project. In this case, the student would have two group accounts associated with their username.
+
+When running a job with either `srun` or `sbatch`, if you have more than one account associated with your username, we recommend you use the `--account=` flag and specify the account that corresponds to the respective project. In the example with a student associated with a class and a student club, if the student is on Discovery submitting a job for a project for their class, set the `account=` flag to the name of the class account. If the student is working on a project for the club, set the `account=` flag to the name of the student club account.
```

Comment:
I can see this section being important if/when we start doing allocations and the user is "charged" for usage from different accounts. But otherwise is using the --account flag just a means of keeping organized? Or is there another reason to use it?

If there's a need to trim down some sections so the overall page is not too long I would recommend removing this.

Reply:
Good call. I removed 80% of this section.